### PR TITLE
CSCQVAIN-191: Fix author and copyright

### PR DESCRIPTION
**Pull request by jppiiroinen**
The correct author and copyright is Ministry of Education and Culture, Finland.

### CODE REVIEW

**Comment by jppiiroinen** [comment]
we should get the upstream changes, which we are waiting as well, but this should be compatible with those.

File: README.md [hunk 1]
```diff
@@ -101,5 +101,5 @@ Note that because Qvain depends on [Vue](https://vuejs.org/v2/guide/installation
 ## License
 
 Qvain metadata description tool
-Copyright (c) 2016–2019 National Library of Finland
+Copyright (c) 2016–2019 Ministry of Culture and Education, Finland.
```
Comment (tahme) [inline README.md]:
```suggestion
Copyright (c) 2016–2019 Ministry of Education and Culture, Finland.
```

File: src/person/testdata.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [
 	{
-		"name": "Wouter Van Hemel",
+		"name": "Teppo Testaaja",
 		"email": "wvh@example.com",
```
Comment (ShreyasDeshpande) [inline src/person/testdata.json]: This email id -intital letters are of wouter?? Do we need to change this??

Comment (jppiiroinen) [reply src/person/testdata.json]: didn't see that one there, I will add some other test address

**Comment by jppiiroinen** [comment]
meh.. I failed to revert changes :D ..another try after lunch

**Comment by jppiiroinen** [comment]
Now it should be ok and a single commit. This does not change the test data as that is a different topic.

**Review by tahme — APPROVED** [review]

**Pull request merged**